posthog/models/action.py

```diff
@@ -23,6 +23,7 @@ class Meta:
 
     name: models.CharField = models.CharField(max_length=400, null=True, blank=True)
     team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+    description: models.CharField = models.CharField(max_length=400, blank=True)
```

Comment:
Right now, we cap the limit to 400 characters, but if the user types over 400 characters, we fail silently on the front-end.

I don't have a strong preference, but I think we should either handle the error + make it clear to the user, or if we don't really want to enforce a max_length, then use `models.TextField` .

Reply:
`models.TextField` is generally what we use for descriptions I think.
Regarding the frontend, at least length validation is simple enough to just slap `maxLength` on the input component.

Reply:
👍 to slapping `maxLength`